examples/embeddings/README.md

```diff
@@ -0,0 +1,284 @@
+# Set up and run locally caikit embeddings server
+
+#### Setting Up Virtual Environment using Python venv
+
+For [(venv)](https://docs.python.org/3/library/venv.html), make sure you are in an activated `venv` when running `python` in the example commands that follow. Use `deactivate` if you want to exit the `venv`.
+
+```shell
+python3 -m venv venv
+source venv/bin/activate
+```
+
+### Models
+
+For this tutorial, you can download [all-MiniLM-L6-v2](https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2), to do that you need to follow the steps to clone and use `git lfs` to get all the models files:
+
+```shell
+# Make sure you have git-lfs installed (https://git-lfs.com)
+git lfs install
+
+git clone https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2
+```
+
+To create a model configuration and artifacts, the best practice is to run the module's bootstrap() and save() methods.  This will:
+
+* Load the model by name (from Hugging Face hub or repository) or from a local directory. The model is loaded using the sentence-transformers library.
+* Save a config.yml which:
+  * Ties the model to the module (with a module_id GUID)
+  * Sets the artifacts_path to the default "artifacts" subdirectory
+  * Saves the model in the artifacts subdirectory
+* Check an example of the folder structure at [models_](./models_/)
+
+> For the reranker service, models supported are bi-encoder and are the same used by the other embeddings tasks.
+
+This can be achieved by the following lines of code, using BGE as example model:
+
+```python
+import os
+os.environ['ALLOW_DOWNLOADS'] = "1"
+
+import caikit_nlp
+model_name = "BAAI/bge-large-en-v1.5"
```

Comment:
Can we change the model name to "sentence-transformers/all-MiniLM-L6-v2" just so that the person who follows this tutorial can mostly copy and paste the commands?